querybook/server/datasources/datadoc.py

```diff
@@ -286,7 +287,13 @@ def create_datadoc_schedule(
             schedule_name,
             "tasks.run_datadoc.run_datadoc",
             cron=cron,
-            kwargs={**kwargs, "user_id": current_user.id, "doc_id": id},
+            kwargs={
+                **kwargs,
+                "user_id": current_user.id,
+                "doc_id": id,
+                "scheduled": True,
+                "execution_type": QueryExecutionType.SCHEDULED.value,
```

Comment:
since its defaulted to `QueryExecutionType.SCHEDULED.value`, don't put the value here?

Reply:
I was thinking we may support to adhocly run an entire datadoc in the future. but you're right, we could just pass `QueryExecutionType.ADHOC` for it and keep the default value in `run_datadoc` task as `scheduled`